lib/ims/lti/models/serializable.rb

```diff
@@ -0,0 +1,12 @@
+module IMS::LTI::Models
+  module Serializable
+    def as_json
```

Comment:
active support expects as_json to be able to take a options hash as an argument, we just add it to keep error from being thrown.  see: http://api.rubyonrails.org/classes/ActiveModel/Serializers/JSON.html